README.md

```diff
@@ -33,15 +34,25 @@ and
 *or if you're using Gradle:*
 
 ```bash
-implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.0'
+implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.1'
 ```
 
 ## Initialize ```RudderClient```
 ```java
+FlushBlock flushBlock = FlushBlock.create(); // Initialization in order to block further method invocation until the flush completes. 
```

Comment:
I think this might be a bit confusing to the user. Can we implement this logic internally inside the SDK?

We can add one more method under `RudderAnalytics.builder()` as `synchronised(true)` and default as `false`

```JAVA
RudderAnalytics analytics = RudderAnalytics.builder("write_key", "data_plane_url")
										.synchronize(true) // optional (default : false)
										.build();
```

The `build` method will initiate the `FlushBlock` internally and will keep it ready for use.